### PR TITLE
Modifying settings

### PR DESCRIPTION
**Pull request by perlajarillo**
Including:

- Expansion panels to incorporate more sections in settings as our project is growing.
- Cancel account functionality.

### CODE REVIEW

File: src/components/Settings/Settings.js [hunk 1]
```diff
@@ -1,13 +1,72 @@
...
+        <Typography variant="headline">Settings</Typography>
+        <br />
+        <br />
+        <ExpansionPanel>
+          <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+            <PermIcon />
+            <Typography className={classes.heading}>Change password</Typography>
+          </ExpansionPanelSummary>
+          <ExpansionPanelDetails>
+            <PasswordChange />
```
Comment (Fa-v) [inline src/components/Settings/Settings.js]: We can add marginRight: theme.spacing.unit * 3 to the icons in this section.

File: src/components/CancelAccount/CancelAccount.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        <Typography>
+          <AlertIcon className={classes.iconBig} />
+          This operation can NOT be undone. If you delete your account you will
+          not be able to use True Relief's tools, like Tracking Pain, Local
+          Providers and Treatments.
+        </Typography>
+        <br />
+        <Button
+          variant="contained"
+          color="secondary"
```
Comment (Fa-v) [inline src/components/CancelAccount/CancelAccount.js]: I know you've already changed this icon's color to red ;)

**Comment by Fa-v** [comment]
Everything is coming together!

File: src/components/Settings/Settings.js (hunk 1 above)
Comment (perlajarillo) [reply src/components/Settings/Settings.js]: Good idea! Done!

**Comment by N8-B** [comment]
Looks good! Great job!

**Comment by N8-B** [comment]
Sorry! Closed the pull request by mistake.

**Pull request merged**